staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -198,7 +198,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	AdvancedAuditing: {Default: true, PreRelease: featuregate.GA},
 
-	CustomResourceValidationExpressions: {Default: false, PreRelease: featuregate.Alpha},
+	CustomResourceValidationExpressions: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
please update the constant to indicate when this moved to beta.

Reply:
Addressed above. Thanks